app/controllers/results_controller.rb

```diff
@@ -335,12 +324,8 @@ def next_grouping
       end
       next_result = Result.find_by(id: next_grouping&.result_id)
     else
-      groupings = assignment.groupings.joins(:group).order('group_name')
-      if params[:direction] == '1'
-        next_grouping = groupings.where('group_name > ?', grouping.group.group_name).first
-      else
-        next_grouping = groupings.where('group_name < ?', grouping.group.group_name).last
-      end
+      reversed = params[:direction] == '1'
```

Comment:
This is the opposite of the expected behaviour

Reply:
Very true. Will fix